params/celo.go

```diff
@@ -2,4 +2,8 @@ package params
 
 const (
 	DefaultGasLimit uint64 = 20000000 // Gas limit of the blocks before BlockchainParams contract is loaded.
+
+	BaklavaIsthmusTimestamp     uint64 = 1598918400 // 2020-09-01T00:00:00Z
+	AlfajoresIsthmusTimestamp   uint64 = 1598918400 // 2020-09-01T00:00:00Z
+	CeloMainnetIsthmusTimestamp uint64 = 1598918400 // 2020-09-01T00:00:00Z
```

Comment:
Please add a comment that these are not the real dates, since they are somewhat plausible.